### ISSUE TITLE
Mac:  VoiceOver issues

### ISSUE THREAD

**Issue by petersampsonaudacity** [audacity/audacity#1546]
**Describe the bug**
There are several issues with Voiceover on macOS - these were originally looged as separate bugs in Bugzilla:

1) Bug 726 Mac: VoiceOver: Tracks not read after exporting
https://bugzilla.audacityteam.org/show_bug.cgi?id=726

2) Bug 1846 Mac: VoiceOver does not read the names of tracks (or Preference categories) when using standard keyboard navigation
https://bugzilla.audacityteam.org/show_bug.cgi?id=1846

3) Bug 1847 - Mac: VoiceOver does not read the names of text boxes
https://bugzilla.audacityteam.org/show_bug.cgi?id=1847

4) Bug 1848 - Mac: VoiceOver does not read the name of choice controls
https://bugzilla.audacityteam.org/show_bug.cgi?id=1848

5) Bug 1892 - Mac: VoiceOver does not read the categories in Preferences using standard keyboard navigation
https://bugzilla.audacityteam.org/show_bug.cgi?id=1892

6) Bug 2474 - Mac: VoiceOver - Keyboard Preferences bindings list not read
https://bugzilla.audacityteam.org/show_bug.cgi?id=2474

7) Bug 2108 - Mac: VoiceOver can't open Combo box menu
https://bugzilla.audacityteam.org/show_bug.cgi?id=2108

8) Bug 2103 - Mac: VoiceOver does not read lists in Manage Macros dialog
https://bugzilla.audacityteam.org/show_bug.cgi?id=2103

9) Bug 2102 - Mac: VoiceOver unable to change keyboard shortcuts
https://bugzilla.audacityteam.org/show_bug.cgi?id=2102

**To Reproduce**
Try using VoiceOver on macOS

See Bugzilla bug threads
**Expected behavior**
VoiceOver to work and read properly on Mac

**Screenshots**

**Additional information (please complete the following information):**
 - OS: macOS
 - Version all Audacity

**Additional context**
None

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
